templates/customernotification.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+    <html>
+        <head>
+            <title>Hello</title>
+        </heah>
+        <body itemscope itemtype="http://schema.org/EmailMessage">
```

Comment:
The `<html>`, `<head>` and `<body>` blocks, should have an indentation of 0 (in accordance with proper HTML formatting)

Also the closing tag of `<head>` is `</head>`

Reply:
Done